monai/visualize/img2tensorboard.py

```diff
@@ -67,18 +73,18 @@ def make_animated_gif_summary(
     image: Union[np.ndarray, torch.Tensor],
     writer=None,
     max_out: int = 3,
-    animation_axes: Sequence[int] = (3,),
-    image_axes: Sequence[int] = (1, 2),
-    other_indices: Optional[Dict] = None,
+    frame_dim: int = -3,
```

Comment:
I feel the previous `image_axes` is more powerful, with the flexibility of doing transpose operation, why removing it? also the docstring still has the option `image_axes`

Reply:
Hi @wyli ,

Thanks for your review and feedback.
I think for most cases, we only need to specify the dim as frames, so I think a `frame_dim` is easier to understand and use, and also align with our `matshow3d` API.
Do you have any use cases that need to specify frame dim and also transpose the left 2 spatial dims? Maybe we don't need to consider this case so far? I suspect very few users used this API directly.
Sorry for the mistake in doc-string, I will update it ASAP.

Thanks.

Reply:
yes, a potential use case: https://github.com/Project-MONAI/tutorials/issues/351

Reply:
OK, thanks for your sharing.
Let me try to update ASAP.

Reply:
Hi @wyli @ericspod ,

I checked again the previous code logic in `make_animated_gif_summary`, if I understand correctly, actually it doesn't have the transpose logic(same as the doc-string said it's not used):
https://github.com/Project-MONAI/MONAI/blob/dev/monai/visualize/img2tensorboard.py#L92
`image = image[tuple(slicing)]` just do redundant logic as all the image axes are in `axis_order`.
Do you mean to implement `transpose` logic to all these visualization functions and `matshow3d`?

Thanks.

Reply:
Hi @ericspod @rijobro ,

What do you guys think about this feature? Should we just use `frame_dim` to specify the dimension as frames, or use `image_axes` and `frame_axis` to powerfully support `transpose` all the axes?

Thanks in advance.